// 1. Handle array index out of bounds exception using try_catch and finally.
// class exception_que {
// public static void main(String[] args) {
// try {
// int[] ar = { 1, 32, 12, 12, 213, 2, 3, 121, 231, 23, 12, 31 };
// int y = ar[20];
// } catch (ArrayIndexOutOfBoundsException e) {
// System.out.println(e);
// }
// }
// }

// 2. Handle null pointer exception using try catch and finally.
// class exception_que {
// public static void main(String[] args) {
// try {
// String ln = null;
// System.out.println(ln);
// } catch (NullPointerException e) {
// System.out.println(e);
// }
// }
// }
// 3. Handle interrupted exception using try catch and finally.